### PR TITLE
fix(card): add missing hasRipple (#248)

### PR DESCRIPTION
**Pull request by bmihelac**

### CODE REVIEW

File: packages/card/PrimaryContent.js [hunk 1]
```diff
@@ -24,33 +24,67 @@ import React from 'react';
...
-      >
-        {children}
-      </div>
+      <PrimaryContentDefault {...otherProps} />
     );
   }
 }
 
+
+const PrimaryContentDefault = (props) => {
```
Comment (moog16) [inline packages/card/PrimaryContent.js]: Instead of PrimaryContentDefault, can we name it PrimaryContentBase?

Comment (bmihelac) [reply packages/card/PrimaryContent.js]: No problem. That should probably be renamed as well in material-icon:

https://github.com/material-components/material-components-web-react/blob/master/packages/material-icon/index.js#L57-L80

**Comment by bmihelac** [comment]
Good question. Looking mdc-card it looks like there is no use-case  for PrimaryContent without ripple:

> Only applicable to cards that have a primary action that the main surface should trigger.

I can refactor this PR to remove `hasRipple` prop and export decorated with `withRipple` component instead.

**Comment by moog16** [comment]
@bmihelac that sounds good to me!

**Comment by bmihelac** [comment]
@moog16 PR has been refactored (would rebase when add tests)

One thing that I am not sure how to do is testing that PrimaryContent has ripple. Here is my not working code:

https://github.com/material-components/material-components-web-react/blob/728a0ee0c3f01a948c74ded4398c1ff44ac12d40/test/unit/card/Content.test.js#L15-L21

Any idea how to test this?

**Comment by moog16** [comment]
@bmihelac figured it out...the upgraded class is added in a requestAnimationFrame. So just wrap the assert in a requestAnimationFrame.

**Comment by moog16** [comment]
@bmihelac - instead of wrapping it in a rAF, please use the rAF helper we have https://github.com/material-components/material-components-web-react/blob/master/test/unit/helpers/raf.js. Example seen in (https://github.com/material-components/material-components-web-react/blob/master/test/unit/ripple/index.test.js#L27)[ripple unit test].

**Comment by bmihelac** [comment]
Thanks @moog16, that worked perfectly.
I have added missing test and rebased PR.

**Comment by bmihelac** [comment]
Let me know if anything else is needed regarding this.

**Comment by moog16** [comment]
looks good to me. I'm going to open another pr for tests

**Comment by moog16** [comment]
Once CI passes on this PR: https://github.com/material-components/material-components-web-react/pull/263 we can merge this PR.

@bmihelac can you please sign this PR?

**Comment by bmihelac** [comment]
> @bmihelac can you please sign this PR?

signed.

**Comment by moog16** [comment]
CI Passing: https://travis-ci.org/material-components/material-components-web-react/builds/430245677?utm_source=github_status&utm_medium=notification

Merging!

**Comment by bmihelac** [comment]
thank you @moog16 !

**Pull request merged**